docs/general/server/media/Lyrics.md

```diff
@@ -0,0 +1,63 @@
+---
+uid: server-media-lyrics
+title: Lyrics
+---
+
+# Lyrics
+Lyrics are contained in the same folder and must match the filenames for their corresponding item. For example: The lyric file for `01 Death Eternal.mp3` must be `01 Death Eternal.lrc` or `01 Death Eternal.txt`.
```

Comment:
We also support `.elrc`:
https://github.com/jellyfin/jellyfin/blob/3738f87278832b0a5995567e5a77d68a171a76cf/Emby.Naming/Common/NamingOptions.cs#L176-L181